src/components/Search/SortingMenu.tsx

```diff
@@ -16,8 +17,12 @@ const SORT_KEY = 'order'
 export default function SortingMenu() {
   const location = useLocation()
   const params = useMemo(() => new URLSearchParams(location.search), [location.search])
-  const order = useMemo(() => (params.get('order') === 'ASC' ? 'ASC' : 'DESC'), [params])
-  const arrowDirection = useMemo(() => (order === 'ASC' ? 'Upwards' : 'Downwards'), [order])
+  const isSearching = useMemo(() => !!params.get('search'), [params])
+  const order = useMemo(
+    () => toSearchSorting(params.get('order')) ?? (isSearching ? 'RELEVANCE' : SortingOrder.DESC),
+    [isSearching, params]
+  )
+  const arrowDirection = useMemo(() => (order === SortingOrder.ASC ? 'Upwards' : 'Downwards'), [order])
```

Comment:
Memo not necessary here (and it's possible that is also not necessary in the other variables)